ChronosTankDrive/src/main/java/frc/robot/Robot.java

```diff
@@ -166,18 +175,17 @@ public void accelLimit(double leftInput, double rightInput) {
   public void chaseBall() {
     NetworkTable table = NetworkTableInstance.getDefault().getTable("photonvision")
         .getSubTable("Microsoft_LifeCam_HD-3000");
-    double angle = table.getEntry("targetYaw").getDouble(-9.0);
-    double turn = -(angle) / 30;
-    double left = turn;
-    double right = -turn;
-    double forward = 0;
-    System.out.println(angle);
-    if (Math.abs(angle) < 7) {
-      left = -.7;
-      right = -.7;
-    }
-    accelLimit(left, right);
-    chronosDrive.tankDrive(leftAdjusted, rightAdjusted);
+    if (!table.getEntry("hasTarget").getBoolean(false))
+      return;
+    double angle = table.getEntry("targetYaw").getDouble(0);
+    double turn = -(angle) / 30 * .5;
+    double distance = 9.5
+        / (Math
+            .tan(Math.sqrt(Math.tan(table.getEntry("targetArea").getDouble(1000000) / 100)) * (54.8 / 180 * Math.PI)))
```

Comment:
Did I type this in wrong, or did something change? This is tan(sqrt(tanA)*54.8) .  There should not be any tanA. I'm surprised this worked if this is what we used.

Reply:
It was present in your commit of this code. Maybe that is why we were off by a factor of two.